src/LibraryUtilities.ts

```diff
@@ -492,13 +489,10 @@ export function buildLibrarySectionsFromLayoutSpecs(loadedTypes: any, layoutSpec
         sectionElements.push(new LayoutElement(section));
     }
 
-    //call update category groups.
-    updateCategoryGroups(sectionElements);
```

Comment:
is this call not necesary? or is done in other place?

Reply:
@RobertGlobant20 Please, let me know if I'm missing something but If you look at the function declaration, it doesn't return anything that might affect the context where it is executed. Also, all the operations there are related to `sectionElements` which is defined as a `let`.

Reply:
yeah looks like the function was disabled so no longer valid